index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Theophilus Doku</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <!-- Header -->
+    <header class="header-wrapper">
+        <a href="#" class="logo">
+            <h1>Sozo</h1>
+        </a>
```

Comment:
- [ ] Great job using an anchor `<a>` tag for this one! 🚀 However, I suggest that you don't use the `<h1>` tag here. Since this is intended to be a logo and not the main title of the page. Please consider editing this one! 🤓 🛠️ 

```suggestion
        <a href="#" class="logo">
            Sozo
        </a>
```

Reply:
Thanks Andre